packages/Babylonian-UI.package/BPBrowser.class/instance/toggleSidePane.st

```diff
@@ -0,0 +1,26 @@
+what to show
+toggleSidePane
+
+	| builder newWindow oldSidePaneWidth |
+	builder := ToolBuilder default.
+		
+	self okToChange ifTrue: [
+		self isSidePaneVisible
+			ifTrue: [sidePaneContent = #none
+				ifTrue:  [
```

Comment:
maybe a dictionary to have more of a switch case struct when comparing symbols?

Reply:
also: long method too, maybe extract the `self isSidePaneVisible` cases?

Reply:
didn't use a dictionary but shortened code and conditions